crt_portal/cts_forms/fixtures/assignee-removed-report.json

```diff
@@ -0,0 +1,282 @@
+[
```

Comment:
Thank you for this fixture! Would appreciate some better fake data though, `"violation_summary": "dfdafsadfasd"` isn't too representative. `"violation_summary": "violation summary here"` would be sufficient.

Reply:
Thank you for updating `violation_summary` but I meant it as an example. The data here is still a bit noisy, in terms of having `asdf` in it, can we make it a bit nicer?